controllers/api/v1/me_test.go

```diff
@@ -0,0 +1,41 @@
+package apiv1
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MeController", func() {
+	Describe("GET /api/v1/me", func() {
```

Comment:
How about creating a health check endpoint [as done by Hoang](https://github.com/hoangmirs/go-scraper/pull/54/files#diff-f274235a2d89036f75c19f1bea39fb17df98bae043b6f46815494a81edf4da49). At least, it's not something you will have to remove later and it's actually useful.

Reply:
Updated in https://github.com/junan/google-scraper/pull/45/commits/82c0c77bf3b8e8211485ad61b3cc7ab48a23d79c